### PR TITLE
script to generate release notes from PT stories

### PR DESCRIPTION
**Pull request by scytacki**
This generates markdown from PT that is similar to 
https://github.com/concord-consortium/dev-templates/blob/main/new-release.md

I'm not sure if this is the right repository for this script, but since the template is here it seemed OK. If you've got other ideas of where to put it please share.

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Scripts to populate templats
```
Comment (kswenson) [inline scripts/README.md:1]:
```suggestion
# Scripts to populate templates
```

File: scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    npm install
+
+## Release Notes
+
+This uses Pivotal Tracker to print out markdown like what is suggested in the `new-release.md` file.
+- searches the Orange and Teal PT boards for stories with a specific label
+- ignores chores and releases
+- strips the `**[label]**` off of the front of the stories
+
+To run this script need your PT token: https://www.pivotaltracker.com/help/articles/api_token/
```
Comment (kswenson) [inline scripts/README.md:14]:
```suggestion
To run this script you need your PT token: https://www.pivotaltracker.com/help/articles/api_token/
```

File: scripts/release-notes.mjs [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    }
+  }
+}
+
+const ptLabel = process.argv[2];
+const search = `label:${ptLabel} includedone:true`;
+// Orange
+await collectStories(2441249, search)
+// Teal
+await collectStories(2441242, search)
```
Comment (kswenson) [inline scripts/release-notes.mjs:40]: Named constants would be more self-explanatory here, e.g. `tealProjectId`.

**Review by dougmartin — APPROVED** [review]
Looks good :+1:

File: scripts/release-notes.mjs (hunk 3 above)
Comment (scytacki) [reply scripts/release-notes.mjs]: Good point

**Pull request merged**